### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -7,504 +7,496 @@ msgid ""
...
+msgid "Other"
+msgstr "Overig"
 
 #. Translators: the {} will be replaced with the button index, e.g.
 #. "Button 1 click".
-#: piper/buttondialog.py:222 piper/buttonspage.py:90
+#: piper/buttondialog.py:257 piper/buttonspage.py:105
 msgid "Button {} click"
-msgstr "Klik met de {}knop"
+msgstr "{}muisklik"
```
Comment (staticssleever668) [inline po/nl.po:90]: Does this sound right? It will be like "1muisklik", "2muisklik" and so on.

**Review by staticssleever668 — COMMENTED** [review]
"DPI" -> "dpi", "LED" -> "led", "API" -> "api", all of these seem like unintended changes

**Comment by Vistaus** [comment]
> "DPI" -> "dpi", "LED" -> "led", "API" -> "api", all of these seem like unintended changes

No, we usually write those terms with non-caps.

File: po/nl.po (hunk 1 above)
Comment (Vistaus) [reply po/nl.po:90]: You're right: I may have interpreted that wrong. Will think of a better translation.

**Review by staticssleever668 — APPROVED** [review]

**Comment by staticssleever668** [comment]
Thank you!

The "advanced" page seems not to be translated, feel free to do it in another PR.
image

**Comment by Vistaus** [comment]
Hmm, I didn't see the advanced page in nl.po or the pot file. Or at least Poeditor didn't show it to me.

**Comment by Vistaus** [comment]
Thanks for mentioning the advanced and for looking into the issue. :)

**Comment by staticssleever668** [comment]
Fixed: #871.

By the way, when I merged your PR, I force-pushed to your branch, so you will have to do a reset `git reset --hard origin/master`.

**Comment by Vistaus** [comment]
Thanks, I'll do that! :)

**Pull request merged**